### ISSUE TITLE
Unable to find suitable version for underscore

### ISSUE THREAD

**Issue by AdamKyle** [bower/bower#1578]
I get the following error:

```
Message:
    Unable to find suitable version for underscore
Details:
    code: ECONFLICT
    picks: [object Object],[object Object],[object Object]
```

With the following bower file, this error I have never come across before. I cannot use the interactive shell, as this gets deployed to continuous integration. We also prefer to use Github repo links (don't ask me why) over bower packages.

```js
{
    "name": "Nightbird",
    "version": "0.0.1",
    "main": "src/css/style.scss",
    "dependencies": {
      "backbone": "git@github.com:jashkenas/backbone.git#1.1.2",
      "underscore": "git@github.com:jashkenas/underscore.git#1.6.0",
      "aisis-bootstrap-theme": "git@github.com:AdamKyle/Aisis-Bootstrap-Theme.git#0.5.0",
      "selectize.js": "git@github.com:brianreavis/selectize.js.git#0.8",
      "underscore.string": "git@github.com:epeli/underscore.string.git#v2.3.2",
      "jquery-bootpag": "git@github.com:botmonster/jquery-bootpag.git#1.0.5",
      "underscore.inflection": "git@github.com:jeremyruppel/underscore.inflection.git",
      "moment": "git@github.com:moment/moment.git",
      "bootstrap-markdown": "git@github.com:toopay/bootstrap-markdown.git#2.5.0",
      "markdown-js": "git@github.com:evilstreak/markdown-js.git#v0.5.0",
      "to-markdown": "git@github.com:domchristie/to-markdown.git#v0.0.2",
      "font-awesome": "git@github.com:FortAwesome/Font-Awesome.git#4.2.0",
      "react-bower": "git@github.com:reactjs/react-bower.git#0.11.1",
      "showdown": "git@github.com:coreyti/showdown.git#0.3.1",
      "pure": "git@github.com:yahoo/pure.git#0.5.0"
    }
}
```

Any idea whats going on? Is this a bug? or just a developer being dumb?

**Comment by sheerun** [comment]
You probably use some software that uses bower underneath and doesn't handle this case. Bower shows interactive selector, just like belhaj mentioned.

Please open an issue on repository of this software.

**Comment by gitmalux** [comment]
Unable to find a suitable version for jquery, please choose one by typing one of the numbers below:
    1) jquery#>=1.9.1 <=2.1.1 which resolved to 2.1.1 and is required by kendo-ui#2016.3.914+Official
    2) jquery#2.2.x which resolved to 2.2.4 and is required by angular-seed
    3) jquery#1.9.1 - 3 which resolved to 3.1.1 and is required by bootstrap#3.3.7
